.github/workflows/package.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           cat .repo.patch
```

Comment:
Just thinking if there are downsides to not having the submodule. One aspect I see is that on each commit it is clear what commit from master was used for making certain changes (https://github.com/mongodb/awscdk-resources-mongodbatlas/pull/321), while now we would not have that information.
Eventually if all our L1 resources are constantly generated and up to date with master this would not be relevant.

Reply:
that's a good point, but yes, that's the idea, to check that they're always synch'ed

Reply:
as an example therer will be a GH action that will fail if resources are not up-to-date

Reply:
thanks for the context 👍